foladipo/docs-cabinet-cp2
# Feature/147831453/create update view delete documents

#### What does this PR do?
Enables users to create, view and delete documents. It also updates the project README.

#### Description of task to be completed
Add ability to the create, view or delete documents.

#### How should this be manually tested?
*Required*: You need to have a Postgres database server running on your computer. Then, create a `.env` according to [this template](https://github.com/andela-foladipo/docs-cabinet-cp2/blob/feature/146115889/create-endpoints-using-tdd/.env.sample).

- Follow the instructions stated in the README to set up your local instance of this app.

- Launch the app in a browser and you should be able to create, view and delete documents.

#### Any background context you want to provide?
N/A

#### What are the relevant Pivotal Tracker stories?
147831453

#### Screenshots
N/A
